lib/run/index.js

```diff
@@ -82,7 +82,8 @@ module.exports = function (options, callback) {
         runner.run(options.collection, {
             stopOnFailure: options.bail, // LOL, you just got trolled ¯\_(ツ)_/¯
             abortOnFailure: options.abortOnFailure,
-            iterationCount: options.iterationCount,
+            iterationCount: options.iterationCount ||
+                (_.get(options, 'iterationData') ? options.iterationData.length : 1),
```

Comment:
Will this not error out if iteration data is undefined or not an array? @czardoz